cockroachcloud/backups-page.md

```diff
@@ -122,9 +200,57 @@ Additionally, you can [back up and restore](../v20.1/backup-and-restore.html) yo
     If you are backing up the data to AWS or GCP, use the `specified` option for the `AUTH` parameter.
     {{site.data.alerts.end}}
 
-1. To [restore](../v20.1/restore.html) to your CockroachCloud cluster:
+1. To [restore](../stable/restore.html) to your CockroachCloud cluster:
 
     {% include copy-clipboard.html %}
     ~~~ sql
     > RESTORE DATABASE example_database FROM 'gs://bucket_name/path_to_backup?AUTH=specified';
     ~~~
+
+## Troubleshooting
+
+### Resolve a database naming conflict
+
+The databases you want to restore cannot have the same name as an existing database in the target cluster. Before you restore a database, verify that the database name is not already in use. To do this, connect to your cluster with the [CockroachDB SQL client](connect-to-your-cluster.html#use-the-cockroachdb-sql-client) and run the following:
```

Comment:
might slightly edit "connect to your cluster" with "connect to the target cluster" for specificity